docs/docs/30-administration/22-backends/40-kubernetes.md

```diff
@@ -195,6 +195,23 @@ backend_options:
 AppArmor syntax follows [KEP-24](https://github.com/kubernetes/enhancements/blob/fddcbb9cbf3df39ded03bad71228265ac6e5215f/keps/sig-node/24-apparmor/README.md).
 :::
 
+### Annotations and labels
+
+You can specify arbitrary `meta.annotations` and `meta.labels` to be set on the Pod definition for a given workflow step using the following configuration:
+
+```yaml
+backend_options:
+  kubernetes:
+    annotations:
+      workflow-group: alpha
+      io.kubernetes.cri-o.Devices: /dev/fuse
+    labels:
+      environment: ci
+      app.kubernetes.io/name: builder
+```
+
+In order to enable this configuration, you need to add the appropriate environment variables to the agent config as shown below.
```

Comment:
```suggestion
In order to enable this configuration, you need to set [WOODPECKER_BACKEND_K8S_POD_ANNOTATIONS_ALLOW_FROM_STEP](#woodpecker_backend_k8s_pod_annotations_allow_from_step) and/or [WOODPECKER_BACKEND_K8S_POD_LABELS_ALLOW_FROM_STEP](woodepcker_backend_k8s_pod_labels_allow_from_step) environment variables to `true`.
```

Reply:
I like referencing the actual env var names here. One thing that I wasn't sure of at first was whether these needed to be set on the agent or the server, so I'll rework this section a bit.

Reply:
JFYI, any backend is about the [agent](https://woodpecker-ci.org/docs/administration/agent-config#woodpecker_backend).
Although, that is fine to me what you explicitly point to it.

Reply:
Good to know, thanks for the explanation.